src/userservice/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2021 Google LLC
+# Copyright 2019 Google LLC
```

Comment:
nit: license should be 2021

Reply:
I updated all touched files to have the correct copyright year.